datastore/emulator.go

```diff
@@ -108,7 +108,12 @@ func Emulator(ctx context.Context) <-chan struct{} {
 	}
 
 	// This is emulating the old `aetest` framework that has a hardcoded "testapp" value for project
-	if err := os.Setenv("GOOGLE_CLOUD_PROJECT", fmt.Sprintf(fakeProjectID)); err != nil {
+	if err := os.Setenv("GOOGLE_CLOUD_PROJECT", fakeProjectID); err != nil {
+		log.Errorf("couldn't set env GOOGLE_CLOUD_PROJECT: %v", err)
```

Comment:
What will happen to the code that follows this? Will everything fail? Should this be a `log.Fatalf` instead?

Reply:
Agreed, changed to Fatal.